components/filecoin/rs/BUILD.gn

```diff
@@ -16,6 +16,13 @@ rust_static_library("rust_lib") {
   ]
   cxx_bindings = [ "src/lib.rs" ]
 
+  visibility = [
+    "//brave/components/filecoin:rust_lib",
+    "//brave/components/filecoin:unit_tests",
+    "//brave/components/ipfs",
```

Comment:
IPFS is using https://github.com/brave/brave-core/blob/dea0a6219b92ebc6d26ced9183edf84362a2d9e1/components/filecoin/rs/src/lib.rs#L37-L39, it would take some time to convert callers to async and the library usage seems trivial, so I keep them as is for now.